### PR TITLE
Copy updates for AZ specific questions

### PR DESCRIPTION
**Pull request by embarnard**
Copy updates based on AZ specific questions section of https://docs.google.com/document/d/1KeZ-QEXQHaVNk0uxKJOyx9Gqc-5FRi1YHfszQfP4ik8/edit

### CODE REVIEW

File: app/views/state_file/questions/name_dob/edit.html.erb [hunk 1]
```diff
@@ -22,7 +28,13 @@
     </div>
     <% if f.object.ask_spouse_name?  %>
       <div class="blue-group">
-        <p><%= t(".spouse_name", last_four_ssn: current_intake.direct_file_data.spouse_ssn&.last(4)) %></p>
+        <p>
+          <% if f.object.ask_primary_dob? %>
```
Comment (bengolder) [inline app/views/state_file/questions/name_dob/edit.html.erb:32]: should this be `.ask_spouse_dob?` instead of `.ask_primary_dob`?

**Review by mpidcock — APPROVED** [review]
LGTM

**Review by bengolder — APPROVED** [review]
Approved with one question that might need to be addressed before merging

File: app/views/state_file/questions/name_dob/edit.html.erb (hunk 1 above)
Comment (embarnard) [reply app/views/state_file/questions/name_dob/edit.html.erb]: @bengolder good catch! fixed it

**Pull request merged**